Dockerfile

```diff
@@ -34,19 +28,29 @@ COPY go.sum go.sum
 RUN go mod download
 
 # Copy the go source
-COPY cmd/ cmd/
-COPY api/ api/
-COPY controllers/ controllers/
-COPY docker docker/
-COPY third_party/ third_party/
-COPY cloudinit/ cloudinit/
-
-# Allow containerd to restart pods by calling /restart.sh (mostly for tilt + fast dev cycles)
-# TODO: Remove this on prod and use a multi-stage build
-COPY third_party/forked/rerun-process-wrapper/start.sh .
-COPY third_party/forked/rerun-process-wrapper/restart.sh .
-
-RUN go install -v ./cmd/manager
-RUN mv /go/bin/manager /manager
-
-ENTRYPOINT ["./start.sh", "/manager"]
+COPY ./ ./
```

Comment:
This is too much, we should only copy in the files needed or greatly modify the .dockerignore file.

Reply:
in CABPK it requested to switch from the list of files to use `./`...
TBH I don't see this important, because then only the executable gets copied in the final image

Reply:
ok, that's fair.

Reply:
@chuckha Removing `pkg/` directory requires to copy the whole repository to avoid breaking builds every time we add a new package